packages/ckeditor5-ui/src/bindings/addkeyboardhandlingforgrid.ts

```diff
@@ -84,4 +82,52 @@ export default function addKeyboardHandlingForGrid(
 			evt.preventDefault();
 		};
 	}
+
+	// Function returning a right index relatively current index.
```

Comment:
The function actually doesn't know, how many elements there are in the grid row. It sees the collection linearly - only knows where it starts and ends. So the more correct description is that it returns a next index, and if the last element is being checked, returns the first index (0).